docs/upgrade/upgrade-guides/upgrade-guide-from-2024.x-to-2025.1/upgrade-guide-from-2024.x-to-2025.1.rst

```diff
@@ -339,34 +322,34 @@ Download and install the old release
                sudo yum remove scylla\*
                sudo yum install scylla
 
-.. note::
-  
-   If you are running a ScyllaDB Enterprise official image (for EC2 AMI, GCP, or Azure), follow the instructions for Ubuntu.
+   .. group-tab:: EC2/GCP/Azure Ubuntu Image
 
-Restore the configuration file
-------------------------------
+      If you’re using the ScyllaDB official image (recommended), see the **Debian/Ubuntu** 
+      tab for upgrade instructions.
 
-.. code:: sh
+      If you’re using your own image and installed ScyllaDB packages for Ubuntu or Debian, 
+      you need to additionally restore the ``scylla-machine-image`` package.
 
-   sudo rm -rf /etc/scylla/scylla.yaml
-   sudo cp -a /etc/scylla/scylla.yaml.backup-src | /etc/scylla/scylla.yaml
+      #. Restore the |SRC_VERSION| packages backed up during the upgrade
+         (see the **Debian/Ubuntu** tab).
+      #. Install:
+
+            .. code-block::
+
+               sudo apt-get update
+               sudo apt-get remove scylla\* -y
+               sudo apt-get install scylla
+               sudo apt-get install scylla-machine-image
```

Comment:
Same

```suggestion
               sudo apt-get install scylla-enterpraise-machine-image
```

Reply:
Why do we need to use scylla-enterprise-machine-image?
We agree to use the Scylla DB image name and use the enterprise as an alias
So both should work; both scylladb is shorter

Reply:
This is the revert procedure, hence we are moving here back to the old version which is scylla-enterprise...